openstack/neutron/templates/etc/_neutron-policy.json.tpl

```diff
@@ -262,12 +262,12 @@
     "get_flavor_service_profile": "rule:context_is_admin",
     "get_auto_allocated_topology": "rule:context_is_network_admin",
 
-    "create_trunk": "rule:context_is_admin",
-    "get_trunk": "rule:context_is_admin",
-    "delete_trunk": "rule:context_is_admin",
-    "get_subports": "rule:context_is_admin",
-    "add_subports": "rule:context_is_admin",
-    "remove_subports": "rule:context_is_admin",
+    "create_trunk": "rule:context_is_network_admin",
+    "get_trunk": "rule:context_is_network_viewer",
```

Comment:
I suggest to use `context_is_viewer`, trunks don't seem to have a direct relation to networks

Reply:
Trunks create a relationship between Ports. A Trunk has a parent port and a list of subports with segmentation information (segmentation type + segmentation id). I think without a being able to view the ports it would not be any help to see the trunks, therefore I would choose the same policy that `get_port` has.

Reply:
Okay ..and vice versa: not seeing the trunks, but the ports, would probably also not make much sense.